src/chunk.c

```diff
@@ -900,7 +900,9 @@ chunk_set_replica_identity(const Chunk *chunk)
 	}
 
 	ts_catalog_database_info_become_owner(ts_catalog_database_info_get(), &sec_ctx);
+	EventTriggerAlterTableStart((Node *) &cmd);
 	AlterTableInternal(chunk->table_id, list_make1(&cmd), false);
```

Comment:
The first argument to AlterTableInternal should be an OID and it indeed looks like `table_id` is taken from the `ObjectAddress` for the chunk. You might want to double-check, but it seems to make no difference for this particular case.